### DISCUSSION TITLE
Is it Possible to label ticks like this ?[Image], if yes then how?

### DISCUSSION THREAD

**Discussion by Aatif-Thinkitive** [patrykandpatrick/vico#372] [Questions]
![image](https://github.com/patrykandpatrick/vico/assets/108254342/30fca078-e5e3-4d46-8ef0-4f9fc9118e42)

**Comment by patrickmichalik** [comment]
Hello! There’s no such feature at the moment, but we’ll be looking into adding it. Please see #339.

**Reply by Shaikh-Aatif** [reply]
sorry i didnt elaborate, here i am talking about the labeling "fantastic", "good", etc is that too impossible?

**Reply by patrickmichalik** [reply]
Ah, got it. Because you seem to have two accounts, I didn’t notice that discussion #339 was started by you. What you’ve described here is possible—please see `AxisValueFormatter`. `AxisItemPlacer.Vertical` may also come in handy. `VerticalAxis` accepts implementations of both these interfaces.
